src/main/java/ca/corefacility/bioinformatics/irida/ria/web/projects/settings/ProjectSettingsRemoteController.java

```diff
@@ -132,4 +136,27 @@ public String getProjectSettingsRemotePage(@PathVariable Long projectId, final M
 		projectControllerUtils.getProjectTemplateDetails(model, principal, project);
 		return "projects/settings/pages/remote";
 	}
+
+	/**
+	 * Gets the remote synchronization {@link Project} settings
+	 *
+	 * @param projectId the ID of the {@link Project} to read
+	 * @return {@link RemoteProjectSettings} object which has the
+	 * remote project synchronization settings
+	 */
+	@RequestMapping("/remote-settings")
+	@PreAuthorize("hasPermission(#projectId, 'canManageLocalProjectSettings')")
```

Comment:
Same here, this should be in an ajax controller

Reply:
Updated in 5bc543eee44a8410455f2cbe94057ef721e15826